keyboards/ecila/rev1/info.json

```diff
@@ -0,0 +1,25 @@
+{
+    "keyboard_name": "ecila/rev1",
+    "maintainer": "mincedshon",
+    "bootloader": "atmel-dfu",
+    "features": {
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true

```

Comment:
```suggestion
        "bootmagic": false,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": true,
        "rgblight": true
```

Reply:
hi zvecr, thank you for making the changes. should bootmagic be true as stated in the previous rules.mk file?